templates/zerver/help/import-from-mattermost.md

```diff
@@ -11,25 +11,97 @@ into an existing Zulip organization.
 ## Import from Mattermost
 
 First, export your data.  The following instructions assume you're
-running Mattermost inside a Docker container:
+running Mattermost inside a Docker container. Replace `<username>` and
+`<server_ip>` with appropriate values accordingly.
 
-1. SSH into your Mattermost app server
+{start_tabs}
+
+{tab|mm-default}
+
+1. SSH into your Mattermost production server.
+
+    ```
+    ssh <username>@><server_ip>
+    ```
+
+2. Navigate to the directory where the Mattermost server is installed.
+   On a default install of Mattermost, the directory is `/opt/mattermost/bin`.
+
+    ```
+    cd /opt/mattermost/bin
+    ```
+
+3. Run the following command to export the data from all teams on the server.
+
+    ```
+    sudo ./mattermost export bulk export.json --all-teams
+    ```
+
+    This will generate `export.json` and possibly an `exported_emoji` directory.
+    The `exported_emoji` folder will only be created if your users had uploaded
+    custom emoji to the Mattermost server.
+
+4. Tar the exported files by running the following command. Remove `exported_emoji/` from
+   the command if that folder is not generated during the export.
+
+    ```
+    tar --transform 's|^|mattermost/|' -czf export.tar.gz exported_emoji/ export.json`
+    ```
+
+5. Now exit out of the Mattermost server.
+
+    `exit`
+
+6. Finally copy the exported tar file from the server to your local computer
+
+    ```
+    scp <username>@<server_ip>:/opt/mattermost/bin/export.tar.gz .
+    ```
+
+{tab|mm-docker}
+
+1. SSH into your Mattermost server running the docker containers.
+
+    ```
+    ssh <username>@><server_ip>
+    ```
 
 2. Run the following command to export the data.
-  `docker exec -it mattermost-docker_app_1 mattermost export bulk export.json --all-teams`
 
-3. This will generate `export.json` and possibly an `exported_emoji`
-   directory inside the **mattermost-docker_app_1** container.  The
-   `exported_emoji` folder will only be created if your users had
-   uploaded custom emoji to the Mattermost server.
+    ```
+    docker exec -it mattermost-docker_app_1 mattermost export bulk data/export.json --all-teams`
+    ```
+
+    This will generate `export.json` and possibly an `exported_emoji`
+    directory inside the **mattermost-docker_app_1** container. The
+    `exported_emoji` folder will only be created if your users had
+    uploaded custom emoji to the Mattermost server.
+
+3. Now CD into the shared volume to which we exported the
+   files.
+
+    ```
+    cd ./mattermost-docker/volumes/app/mattermost/data/`
+    ```
+
+4. Tar the exported files by running the following command. Remove `exported_emoji/` from
+   the command if that folder is not generated during the export.
+
+    ```
+    tar --transform 's|^|mattermost/|' -czf export.tar.gz exported_emoji/ export.json`
+    ```
+
+5. Now exit out of the Mattermost server.
+
+    `exit`
```

Comment:
I think we can (1) make these commands tarting from step 2 all be a single step with 4 commands to copy-paste; we don't really need to spell out the other details and (2) do a `mkdir -p` somewhere in the middle to ensure `exported_emoji/` always exists, so the `tar` command just works.

With those changes, this can be a mostly copy-pasteable series of commands.